internal/transport/grpchttp2/framer.go

```diff
@@ -53,6 +53,10 @@ const (
 	FlagContinuationEndHeaders Flag = 0x4
 )
 
+func (f Flag) Has(flag Flag) bool {
```

Comment:
Hmm.... wondering is `Contains` would be a better name for this method? `Has` seems too short. 

Also, please add a docstring for this. Thanks.

Reply:
IsSet perhaps?